packages/features/bookings/components/FiltersContainer.tsx

```diff
@@ -92,51 +82,40 @@ const PeopleFilter = () => {
 };
 
 export function FiltersContainer() {
-  const { t } = useLocale();
+  const isFiltersVisible = useBookingMultiFilterStore((state) => state.isFiltersVisible, shallow);
 
-  const [addFilterOptions, toggleOption, isFilterActive] = useBookingMultiFilterStore((state) => [
-    state.addFilterOptions,
-    state.toggleOption,
-    state.isFilterActive,
-    shallow,
-  ]);
+  return isFiltersVisible ? (
+    <div className="no-scrollbar flex w-full space-x-2 overflow-x-scroll rtl:space-x-reverse">
+      <PeopleFilter />
+      <EventTypeFilter />
+      <TeamsFilter />
+    </div>
+  ) : null;
+}
```

Comment:
NIT: we can use auto animate for animation here whenever filters are visible.

Reply:
Used useAutoAnimate